### PR TITLE
Add ru localization for "Try it"

### PR DESCRIPTION
**Pull request by myshov**
## Summary

Related to https://github.com/mdn/yari/pull/5874 and https://github.com/mdn/yari/issues/5868

Adds ru localization for ELS new heading in EmbedInteractiveExample "Try it"

### Problem

Missing `ru` string which was not localizable, thus mixing localized content with English

### Solution

added a `ru` string which was not localizable, thus mixing localized content with English

### CODE REVIEW

**Comment by myshov** [comment]
Ping @schalkneethling for review :)

File: kumascript/macros/EmbedInteractiveExample.ejs [hunk 1]
```diff
@@ -32,6 +32,9 @@ const text = mdn.localStringMap({
   },
   "fr": {
     "title": "Exemple interactif",
+  },
+  "ru": {
+    "title": "Попробуйте в действии",
```
Comment (lex111) [inline kumascript/macros/EmbedInteractiveExample.ejs:37]: Может лучше не использовать глагол в заголовке? Что-нибудь вроде "интерактивный пример"?

Comment (myshov) [reply kumascript/macros/EmbedInteractiveExample.ejs:37]: Хорошая идея

Comment (myshov) [inline kumascript/macros/EmbedInteractiveExample.ejs:37]:
```suggestion
    "title": "Интерактивный пример",
```

**Review by lex111 — APPROVED** [review]
:+1:

**Comment by schalkneethling** [comment]
Thank you for the PR @myshov. @lex111; purely to satisfy my curiosity, why did you decide to go with "interactive example" in the end?

**Comment by lex111** [comment]
@schalkneethling usually it is better not to use verbs in headings, so for the sake of consistency in the translation, I suggested to stick to this recommendation.

**Comment by schalkneethling** [comment]
> @schalkneethling usually it is better not to use verbs in headings, so for the sake of consistency in the translation, I suggested to stick to this recommendation.

Thank you for the clarification, @lex111

**Pull request merged**